src/test/java/gov/loc/repository/bagit/util/PathUtilsTest.java

```diff
@@ -38,22 +38,43 @@ public void testDecode(){
   
   @Test
   public void testEncode(){
-    //just carriage return
-    Path testPath = Paths.get("/foo/bar/ham\r");
-    String expectedEncoded = "/foo/bar/ham%0D";
-    String actualEncoded = PathUtils.encodeFilename(testPath);
-    assertEquals(expectedEncoded, actualEncoded);
-    
-    //just new line
-    testPath = Paths.get("/foo/bar/ham\n");
-    expectedEncoded = "/foo/bar/ham%0A";
-    actualEncoded = PathUtils.encodeFilename(testPath);
-    assertEquals(expectedEncoded, actualEncoded);
-    
-    //both carriage return and new line
-    testPath = Paths.get("/foo/bar/ham\r\n");
-    expectedEncoded = "/foo/bar/ham%0D%0A";
-    actualEncoded = PathUtils.encodeFilename(testPath);
-    assertEquals(expectedEncoded, actualEncoded);
+	if (System.getProperty("os.name").contains("Windows")){
+	    //just carriage return
+	    String testPath = "\\foo\\bar\\ham\r";
```

Comment:
I'm hesitant to have quite so much duplicate code in a test – it seems too easy for someone to forget to update one branch in the future. Since bagit-java already requires Java 8, I was wondering whether this could be unified using java.nio.file.Path or at least `File.separator`